config/default/provisonercapability/disk-csi-qingcloud-com.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: storage.kubesphere.io/v1alpha1
```

Comment:
I suspect this file and the below neonsan-csi-qingstor-com.yaml will be used by ks-installer to preinstall the provisonercapabilities, if so, be sure to submit related PR to ks-installer or talk with @pixiake

Reply:
Yes, that's true, I'll do that when this PR is approved